### PR TITLE
Fixed a few more typos

### PR DESCRIPTION
**Pull request by pentamassiv**

### CODE REVIEW

File: src/components.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ For components we also need to implement the `Model` trait. The `Components` typ
 {{#include ../examples/components.rs:header_model_impl }}
 ```
 
-The update function is rather minimal. If our header bar was more complex, storing state in this component would make sense, but because we just handle a few buttons, we can simply forward messages. For that we can use the `parent_sender`. You can see that the message type of the main application is `AppMsg` and that there's an enum `AppMode`. Both were not introduced yet, but will be explained later. For now, we just need to know that this component will send `SetMode` messages to the app.
+The update function is rather minimal. If our header bar were more complex, storing state in this component would make sense, but because we just handle a few buttons, we can simply forward messages. For that we can use the `parent_sender`. You can see that the message type of the main application is `AppMsg` and that there's an enum `AppMode`. Both were not introduced yet, but will be explained later. For now, we just need to know that this component will send `SetMode` messages to the app.
```
Comment (AaronErhardt) [inline src/components.md:55]: I think this one was correct, though.

**Review by AaronErhardt — CHANGES_REQUESTED** [review]
Thanks a lot! :+1:

You've really found a lot of typos, I'm glad that they can be fixed now.

File: src/components.md (hunk 1 above)
Comment (pentamassiv) [reply src/components.md:55]: According to [this](https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/), I believe it should be 'were' but English is also not my native language so I could be wrong.

Comment (AaronErhardt) [reply src/components.md:55]: Looks like PR taught me something I haven't even learned at school :laughing: 

Thanks!

**Pull request merged**